core/schema/query.graphqls

```diff
@@ -1,3 +1,7 @@
+scalar Void
+
+scalar JSON
```

Comment:
Adding a `JSON` scalar seems to be a pretty common pattern in the graphql ecosystem. It's useful when you need to pass polymorphic data around and want to explicitly indicate to clients the format of the data.

The implementation I added represents the data a string, which is set to the JSON serialization of the actual object (mentioning this specifically because the popular JSON scalar implementations in js/ts are a bit fancier, but I think that's because they are privileged in that they are using "javascript object notation" :-) )